src/PrivateDns/PrivateDns/help/New-AzPrivateDnsZone.md

```diff
@@ -0,0 +1,163 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.PrivateDns.dll-Help.xml
+Module Name: Az.PrivateDns
+online version: https://docs.microsoft.com/en-us/powershell/module/az.privatedns/new-azprivatednszone
+schema: 2.0.0
+---
+
+# New-AzPrivateDnsZone
+
+## SYNOPSIS
+Creates a new private DNS zone.
+
+## SYNTAX
+
+```
+New-AzPrivateDnsZone -Name <String> -ResourceGroupName <String> [-Tags <Hashtable>]
```

Comment:
This is still "Tags" - needs to be regened.

Reply:
done.